android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/vitalsigns/VitalSignsDetailsViewModel.kt

```diff
@@ -84,6 +84,36 @@ constructor(val patientRepository: PatientRepository, var dispatcher: Dispatcher
       patientVitalItem.spO2 = observationValueOrDefault(listObservationspO2)
       patientVitalItem.spO2Unit = listObservationspO2.valueQuantity.unit ?: ""
 
+      /*if (patientVitalItem.height.isNotEmpty() &&
+          patientVitalItem.weight.isNotEmpty() &&
+          patientVitalItem.height.toDouble() > 0 &&
+          patientVitalItem.weight.toDouble() > 0
+      ) {
+        if (patientVitalItem.weightUnit.isNotEmpty()) {
+          if (patientVitalItem.weightUnit.equals("kg", true)) {
+            patientVitalItem.bmi =
+              computeBMIViaMetricUnits(
+                  patientVitalItem.height.toDouble(),
+                  patientVitalItem.weight.toDouble()
+                )
+                .toString()
+            patientVitalItem.bmiUnit = "kg/m2"
+          } else {
+            patientVitalItem.bmi =
+              computeBMIViaUSCUnits(
+                  patientVitalItem.height.toDouble(),
+                  patientVitalItem.weight.toDouble()
+                )
+                .toString()
+            patientVitalItem.bmiUnit = "lbs/in2"
+          }
+        }
+      }*/
```

Comment:
Will this code be needed in future? If so add a descriptive `TODO` otherwise the commented code can be deleted.

Reply:
updated